evergreen.yml

```diff
@@ -1674,6 +1741,26 @@ functions:
         content_type: text/plain
         bucket: mciuploads
         permissions: public-read
+  
+  "publish static code analysis":
```

Comment:
The publishing tasks shouldn't happen if the release fails or we might have failed or non-existent releases files floating around, which seems a little bit weird, no?

Reply:
yeah, that makes sense. I just split the artifact publishing into two tasks, similar to JDBC, informed by our convo on slack.